<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(54, 54, 54)</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>59</y>
     <width>481</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="chatLog">
      <property name="styleSheet">
       <string notr="true">color:rgb(235, 235, 235)</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="userInput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>409</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(229, 229, 229); font-size: 16pt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 186, 186)</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnectButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 186, 186)</string>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>130</y>
     <width>141</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTextEdit" name="activeUsers">
      <property name="styleSheet">
       <string notr="true">color:rgb(235, 235, 235)</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(218, 120, 205); font-size:18pt;</string>
   </property>
   <property name="text">
    <string>Secure Chat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(218, 120, 205); font-size:12pt;</string>
   </property>
   <property name="text">
    <string>Active Users:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnectButton_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(186, 186, 186)</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
